src/Validation/MailValidator.php

```diff
@@ -1,14 +1,18 @@
 <?php
 
-namespace WMDE\Fundraising\Frontend;
+namespace WMDE\Fundraising\Frontend\Validation;
+
+use WMDE\Fundraising\Frontend\MailAddress;
 
 /**
  * @licence GNU GPL v2+
  * @author Christoph Fischer < christoph.fischer@wikimedia.de >
+ * @author Gabriel Birke < gabriel.birke@wikimedia.de >
  */
-class MailValidator {
+class MailValidator implements ScalarValueValidator {
```

Comment:
I'd suggest that we either allow empty values and add the `ValidationConstraint` `RequiredFieldValidator` to the `RequestValidator` (or any other validator we might want to implement), or provide both an `OptionalMailValidator` and a `RequiredMailValidator`.